packages/framework/fluid-static/src/types.ts

```diff
@@ -9,7 +9,10 @@ import type { SharedObjectKind } from "@fluidframework/shared-object-base";
 import type { ISharedObjectKind } from "@fluidframework/shared-object-base/internal";
 
 /**
- * Valid compatibility modes that may be specified when creating a DOProviderContainerRuntimeFactory.
+ * Determines the set of runtime options that Fluid Framework will use when running.
```

Comment:
Note: this doc for public API surface mentioned `DOProviderContainerRuntimeFactory` which is not even exported from the module where it is defined. I think the new docs (which used to be on `compatibilityModeRuntimeOptions`, also not exported) are good for the public API.